conf/distro/nilrt.conf

```diff
@@ -25,3 +25,23 @@ KERNEL_IMAGEDEST = "boot/runmode"
 ROOT_HOME = "/home/admin"
 
 IMAGE_LINGUAS ?= "en-us en-us.iso-8859-1"
+
+BUILD_IMAGES_FROM_FEEDS = "1"
+
+# Get NIOE packages from local deploy feed and NI packages from web feed
+IPK_NIOE_SUBFEED_URI ?= "file://${DEPLOY_DIR_IPK}"
+
+FEED_SRC ?= "download.ni.com/ni-linux-rt/feeds"
```

Comment:
Could you name this something like `NI_FEEDS_URI` to keep it closer to the extant `NILRT_FEEDS_URI` variable? Or is this variable a part of the BUILD_FROM_FEEDS feature?

Reply:
I've moved all of these lines to nilrt.inc and reused the NILRT_FEEDS_URI variable.

Reply:
That works too.